laravel/app/AutoAssignment.php

```diff
@@ -217,9 +215,7 @@ private static function handleAssignment($schedule, $resident, $date, $ticketsTo
         // if resident has an anesthesiologist pref - check if anest can be assigned
         if(!is_null($anestPref)){
             // get room
-            $room = ScheduleData::where('date', $date)
-                        ->where('id', $schedule)
-                        ->value('room');
+            $room = ScheduleData::where('date', $date)->where('id', $schedule)->value('room');
```

Comment:
Can this be a `find`? Isn't an `id` enough to uniquely identify a `ScheduleData`?